### PR TITLE
Matt/compression

### PR DESCRIPTION
**Pull request by mattgotteiner**

### CODE REVIEW

File: demo-dotnet/QuantizationAndStorageOptions/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+page_type: sample
+languages:
+  - python
```
Comment (HeidiSteen) [inline demo-dotnet/QuantizationAndStorageOptions/README.md:4]: line 4 -   - csharp

File: demo-dotnet/QuantizationAndStorageOptions/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+name: Vector quantization and storage options in C#
+products:
+  - azure
+  - azure-ai-search
+description: |
+  Using Azure.Search.Documents and the Azure SDK for C#, save on storage when using vector quantization
```
Comment (HeidiSteen) [inline demo-dotnet/QuantizationAndStorageOptions/README.md:10]: line 10 -- add a period, also isn't it Azure SDK for .NET?

File: demo-dotnet/QuantizationAndStorageOptions/README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+urlFragment: quantization-storage-c#
```
Comment (HeidiSteen) [inline demo-dotnet/QuantizationAndStorageOptions/README.md:11]: line 11 -- can you have a pound sign in a URL?  maybe this should be csharp

File: demo-dotnet/QuantizationAndStorageOptions/README.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+---
+
+# Vector quantization and storage options using C# (Azure AI Search)  
+
+The C# program creates vectorized data on Azure AI Search and demonstrates how to save on storage using the following features
```
Comment (HeidiSteen) [inline demo-dotnet/QuantizationAndStorageOptions/README.md:16]: line 16 -- needs  colon

File: demo-dotnet/QuantizationAndStorageOptions/README.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+- Quantization
```
Comment (HeidiSteen) [inline demo-dotnet/QuantizationAndStorageOptions/README.md:18]: line 18 (if it's not too wordy) - Built-in scalar quantization that reduces vector index size in memory and on disk.

I like saying "built-in" because it clarifies that AI Search is providing the quantization and it conveys that this task is going to be easy.

File: demo-dotnet/QuantizationAndStorageOptions/README.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+- Disabling storage of vectors returned in query responses. These vectors are stored separately from vectors used for the queries themselves.
+- Smaller data types than `Edm.Single`.
+
+The sample data is a JSON file containing a pre-chunked version of the sample documents. They have been embedded using text-embedding-3-large with 3072 dimensions.
```
Comment (HeidiSteen) [inline demo-dotnet/QuantizationAndStorageOptions/README.md:22]: line 22 -- can you say anything more about the sample data? Is it hotels, or ???

File: demo-dotnet/QuantizationAndStorageOptions/README.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+You can use [Visual Studio](https://visualstudio.microsoft.com/) or [Visual Studio Code with the C# extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp) for these demos.  
+
+## Run the code
+
+Before running the code, ensure you have the .NET SDK installed on your machine.
+
+1. Clone this repository.  
+
+1. Create a `local.settings.json` file in the same directory as the code that follows the same pattern as `local.settings-sample.json`
```
Comment (HeidiSteen) [inline demo-dotnet/QuantizationAndStorageOptions/README.md:40]: line 40 -- needs a period.

File: demo-python/code/vector-quantization-and-storage/README.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
+---
+page_type: sample
+languages:
+  - python
+name: Vector quantization and storage options in Python
+products:
+  - azure
+  - azure-ai-search
+description: |
+  Using azure-search-documents and the Azure SDK for Python, save on storage when using vector quantization
```
Comment (HeidiSteen) [inline demo-python/code/vector-quantization-and-storage/README.md:10]: line 10, needs a period

File: demo-python/code/vector-quantization-and-storage/README.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+urlFragment: quantization-storage-python
+---
+
+# Vector quantization and storage options using Python (Azure AI Search)  
+
+The Python notebook creates vectorized data on Azure AI Search and demonstrates how to save on storage using the following features
+
+- Quantization
```
Comment (HeidiSteen) [inline demo-python/code/vector-quantization-and-storage/README.md:18]: line 18, if you took the edit for C#, the same edit here is "Built-in scalar quantization that reduces vector index size in memory and on disk."

File: demo-python/code/vector-quantization-and-storage/vector-quantization-and-storage.ipynb [hunk 10]
```diff
@@ -0,0 +1,346 @@
...
+    "print(\"Uploaded embeddings\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Check storage sizes\n",
+    "\n",
+    "Find the new storage size in mb to demonstrate how the various options affect storage"
```
Comment (HeidiSteen) [inline demo-python/code/vector-quantization-and-storage/vector-quantization-and-storage.ipynb:268]: line 268 MB in caps, add a period

**Review by HeidiSteen — APPROVED** [review]
Looks great!  I have minor suggestions for punctuation etc.

File: demo-dotnet/QuantizationAndStorageOptions/README.md (hunk 6 above)
Comment (mattgotteiner) [reply demo-dotnet/QuantizationAndStorageOptions/README.md:22]: contoso
